packages/@uppy/vue/README.md

```diff
@@ -0,0 +1,64 @@
+# @uppy/vue
+
+<img src="https://uppy.io/images/logos/uppy-dog-head-arrow.svg" width="120" alt="Uppy logo: a superman puppy in a pink suit" align="right">
+
+<a href="https://www.npmjs.com/package/@uppy/vue"><img src="https://img.shields.io/npm/v/@uppy/vue.svg?style=flat-square"></a>
+<a href="https://travis-ci.org/transloadit/uppy"><img src="https://img.shields.io/travis/transloadit/uppy/master.svg?style=flat-square" alt="Build Status"></a>
+
+Vue component wrappers around Uppy's officially maintained UI plugins.
+
+Uppy is being developed by the folks at [Transloadit](https://transloadit.com), a versatile file encoding service.
+
+## Example
+
+```vue
+<template>
+  <dashboard-modal 
```

Comment:
Should this be `DashboardModal`?

Reply:
Vue follows the conventions of each language, and in this case, HTML uses kebab case, although camel case works, I'm pretty sure kebab-case is recommended.

> In most projects, component names should always be PascalCase in single-file components and string templates - but kebab-case in DOM templates.
\- [Vue Style Guide](https://vuejs.org/v2/style-guide/#Component-name-casing-in-templates-strongly-recommended)